### PR TITLE
debug access control test

### PR DESCRIPTION
**Pull request by richm**
debug access control test

### CODE REVIEW

**Comment by richm** [comment]
origin flake
/test json-file

File: hack/testing/util.sh [hunk 1]
```diff
@@ -199,31 +199,55 @@ function wait_for_fluentd_to_catch_up() {
...
         os::log::error $FUNCNAME: not found $expected record project logging for \'$fullmsg\' after $timeout seconds
         os::log::debug "$( curl_es ${es_pod} /project.logging.*/_search -X POST -d "$qs" )"
-        os::log::error "Checking journal for '$fullmsg' ..."
-        if sudo journalctl | grep -q "$fullmsg" ; then
-            os::log::error "Found '$fullmsg' in journal"
-            os::log::debug "$( sudo journalctl | grep "$fullmsg" )"
-        elif sudo grep -q "$fullmsg" /var/log/containers/* ; then
-            os::log::error "Found '$fullmsg' in /var/log/containers/*"
-            os::log::debug "$( sudo grep -q "$fullmsg" /var/log/containers/* )"
+        if [ -s $ARTIFACT_DIR/es_out.txt ] ; then
```
Comment (ewolinetz) [inline hack/testing/util.sh]: both the `if`and `else` have a `os::log::error` is this intentional?

File: hack/testing/util.sh [hunk 2]
```diff
@@ -199,31 +199,55 @@ function wait_for_fluentd_to_catch_up() {
...
+            os::log::error "$( cat $ARTIFACT_DIR/es_out.txt )"
         else
-            os::log::error "Unable to find '$fullmsg' in journal or /var/log/containers/*"
+            os::log::error apps record for "$fullmsg" not found in source
         fi
+        if sudo test -f /var/log/es-containers.log.pos ; then
+            os::log::error here are the current container log positions
```
Comment (ewolinetz) [inline hack/testing/util.sh]: error level?

**Comment by ewolinetz** [comment]
other than my questions on log level lgtm

File: hack/testing/util.sh (hunk 1 above)
Comment (richm) [reply hack/testing/util.sh]: yes, because both of these are in the path of the error case

File: hack/testing/util.sh (hunk 2 above)
Comment (richm) [reply hack/testing/util.sh]: because this is in the error handling section

**Comment by richm** [comment]
wow - logging failed right away - es didn't even deploy
/test logging

**Pull request merged**